sci-physics/lhapdf/lhapdf-6.5.4-r3.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
```

Comment:
We don't have to do a revision bump to change the `PYTHON_COMPAT`. Things like the python targets are exceptions to the rule that a `USE` flag and/or `RDEPEND` modification require a revision bump. Because we can be certain that there is no change in what is installed to the disk, and the python targets are usually disabled/enabled on a global basis.

Reply:
The revision bump was more for the new patch and closing the reported bug. Though it could be argued that a `CXXFLAGS` is rarely empty and thus the patch has no impact for most users.

Reply:
The patch is for a build time only issue right? Then users who ran into the problem would already have worked around it. I don't think it will meaningfully change what is installed so then I think we can still skip the revision bump.